tools/NuGet/DynamoVisualProgramming.ZeroTouchLibrary/ZeroTouchLibrary.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>DynamoVisualProgramming.ZeroTouchLibrary</id>
-    <version>1.0.0-beta2</version>
+    <version>1.0.0-beta3</version>
```

Comment:
What if we make this dependent on DynamoServices version 1.0.0?? Will it still work?